ui/components/ui/form-field/form-field.js

```diff
@@ -97,6 +99,7 @@ export default function FormField({
             autoFocus={autoFocus}
             disabled={disabled}
             data-testid={dataTestId}
+            placeholder={placeholder}
```

Comment:
Could you please include this proptype in the story `argTypes:` in `ui/components/ui/form-field/form-field.stories.js`